src/Microsoft.AspNet.Razor.Runtime/TagHelpers/HtmlAttributeNotBoundAttribute.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNet.Razor.Runtime.TagHelpers
+{
+    /// <summary>
+    /// Indicates the associated <see cref="ITagHelper"/> property should not be bound to HTML attributes.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Property, AllowMultiple = false, Inherited = false)]
+    public sealed class HtmlAttributeNotBoundAttribute : Attribute
```

Comment:
I liked `HtmlAttributeNotBound` because when typing the other attribute that modifies TagHelper` properties, `HtmlAttributeName`, you also end up with the `HtmlAttributeNotBound` as an option via IntelliSense.